src/api-service/__app__/node/__init__.py

```diff
@@ -50,6 +50,10 @@ def post(req: func.HttpRequest) -> func.HttpResponse:
     if isinstance(request, Error):
         return not_ok(request, context="NodeUpdate")
 
+    answer = check_require_admins(req)
```

Comment:
Should we even be trying to parse the request if it doesn't have the right perms?

Reply:
I would think we'd rather verify that the request is valid in the first place. This is the convention used across our endpoints.